cmd/revproxy/main.go

```diff
@@ -117,13 +132,16 @@ func main() {
 	}
 	quit := make(chan os.Signal, 1)
 	signal.Notify(quit, os.Interrupt)
-	<-quit  // Wait for interrupt signal from OS
+	<-quit // Wait for interrupt signal from OS
 	// Start shutting down servers
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
 	if err := e.Shutdown(ctx); err != nil {
 		e.Logger.Fatal(err)
 	}
+	if config.Sentry.Enabled {
+		defer sentry.Flush(2 * time.Second)
+	}
```

Comment:
I think it is nicer if you move this right below the command that initializes Sentry (i.e. line 113). Because as it stands right now if we hit the line that says `e.Logger.Fatal(err)` immediately above then this will never run. I already made a suggestion with the edit.